backend/app/main.py

```
@@ -0,0 +1,52 @@
+    allow_origins=["*"],  # Allows all origins
+    allow_credentials=True,
+    allow_methods=["*"],  # Allows all methods
+    allow_headers=["*"],  # Allows all headers
+)
+
+# Mount static files (e.g., JS, CSS)
+# app.mount("/scripts", StaticFiles(directory="scripts"), name="scripts")
+
+# Root endpoint
+@app.get("/", tags=["Root"])
+async def read_root():
```

Ruff (0.8.2)

Missing return type annotation for public function `read_root`

(ANN201)